src/cmake.mk

```diff
@@ -8,8 +8,7 @@ $(PKG)_CHECKSUM := 92c83ad8a4fd6224cf6319a60b399854f55b38ebe9d297c942408b792b1a9
 $(PKG)_SUBDIR   := cmake-$($(PKG)_VERSION)
 $(PKG)_FILE     := cmake-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := http://www.cmake.org/files/v$(call SHORT_PKG_VERSION,$(PKG))/$($(PKG)_FILE)
-$(PKG)_TARGETS  := $(BUILD)
-$(PKG)_DEPS     :=
+$(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)
```

Comment:
> Why do you need this change?

When I first added cmake as a dependency to vtk7 it didn't work. But now I try to reproduce the error and it's gone, so I guess this change isn't needed at all.